static/src/inline-svgs/acquisitions/apple-pay-mark.svg

```diff
@@ -0,0 +1,76 @@
+		c-0.5494,0-1.0999,0-1.6593-0.0043c-0.4561-0.0021-0.9112-0.0082-1.3512-0.0182c-0.7436-0.0201-1.7095-0.0632-2.5834-0.2193
+		c-0.7497-0.1348-1.3782-0.3402-1.9858-0.6503c-0.5979-0.3032-1.1422-0.6988-1.6223-1.1797
+		c-0.4764-0.4756-0.8723-1.0207-1.1784-1.6232c-0.3064-0.6019-0.5114-1.2305-0.6462-1.9852
+		c-0.1558-0.8626-0.1986-1.7874-0.22-2.5777c-0.0122-0.4525-0.0173-0.9049-0.0202-1.3547l-0.0022-1.3279l0.0001-0.3506V14.8242
+		l-0.0001-0.3506l0.0021-1.3251c0.003-0.4525,0.0081-0.9049,0.0203-1.357c0.0214-0.7911,0.0642-1.7163,0.2213-2.5861
+		C3.9094,8.4575,4.1143,7.8289,4.4223,7.224C4.726,6.6261,5.1226,6.0803,5.6015,5.6015c0.477-0.4772,1.0231-0.8739,1.6248-1.1799
+		C7.8274,4.1155,8.4571,3.91,9.2068,3.7751c0.8638-0.1552,1.7909-0.198,2.5849-0.2195c0.4526-0.0123,0.9052-0.0172,1.3544-0.0203
+		l1.6771-0.0033H150.6981"/>
+	<g id="XMLID_34_">
+		<g id="XMLID_44_">
+			<path id="XMLID_46_" d="M43.5084,35.7697c1.4032-1.755,2.3554-4.1116,2.1043-6.5197c-2.0541,0.1022-4.5606,1.3551-6.0118,3.1116
+				c-1.303,1.5041-2.4563,3.9593-2.1557,6.2665C39.751,38.8281,42.0547,37.4756,43.5084,35.7697"/>
+			<path id="XMLID_45_" d="M45.5865,39.0786c-3.3486-0.1995-6.1956,1.9004-7.7948,1.9004c-1.5999,0-4.0487-1.7999-6.6972-1.7514
+				c-3.4472,0.0506-6.6458,1.9997-8.3952,5.0996c-3.598,6.2015-0.9495,15.4004,2.5494,20.4511
+				c1.6992,2.4988,3.7469,5.2501,6.4452,5.1512c2.5494-0.1,3.5486-1.6507,6.6475-1.6507c3.0966,0,3.9967,1.6507,6.6954,1.6007
+				c2.7986-0.05,4.5482-2.5,6.2474-5.0011c1.9492-2.8485,2.7471-5.5989,2.7973-5.7499c-0.0502-0.05-5.3964-2.101-5.446-8.2509
+				c-0.0505-5.1494,4.1974-7.5987,4.3973-7.7506C50.634,39.5791,46.8859,39.1791,45.5865,39.0786"/>
+		</g>
+		<g id="XMLID_35_">
+			<path id="XMLID_41_" d="M78.9732,32.1102c7.278,0,12.3464,5.0168,12.3464,12.3209c0,7.3302-5.1722,12.3733-12.5284,12.3733
+				H70.733v12.8142h-5.8225V32.1102H78.9732z M70.733,51.9172h6.6804c5.0689,0,7.9538-2.729,7.9538-7.46
+				c0-4.7305-2.8849-7.434-7.9278-7.434H70.733V51.9172z"/>
+			<path id="XMLID_38_" d="M92.7641,61.8472c0-4.8092,3.6651-7.5645,10.4231-7.9801l7.252-0.4423v-2.0792
+				c0-3.0413-2.0015-4.7049-5.5623-4.7049c-2.9376,0-5.069,1.5076-5.5107,3.821h-5.2509c0.1564-4.8609,4.731-8.3956,10.9175-8.3956
+				c6.6543,0,10.9952,3.4831,10.9952,8.8894v18.6631h-5.3808v-4.4964h-0.1298c-1.5337,2.9371-4.913,4.7822-8.5781,4.7822
+				C96.5329,69.9044,92.7641,66.6815,92.7641,61.8472z M110.4392,59.4296v-2.1058l-6.4723,0.4161
+				c-3.639,0.2337-5.5362,1.5854-5.5362,3.9509c0,2.2873,1.9754,3.7694,5.0684,3.7694
+				C107.4499,65.4602,110.4392,62.9382,110.4392,59.4296z"/>
+			<path id="XMLID_36_" d="M120.9746,79.6522v-4.4964c0.364,0.0512,1.2475,0.1033,1.7152,0.1033
+				c2.5736,0,4.0291-1.091,4.9131-3.8987l0.5199-1.6636l-9.8516-27.2928h6.0822l6.8624,22.1457h0.1298l6.8624-22.1457h5.9268
+				l-10.2156,28.6706c-2.3394,6.5761-5.0168,8.734-10.6834,8.734C122.7941,79.8086,121.3642,79.7565,120.9746,79.6522z"/>
+		</g>
+	</g>
+</g>
```

Comment:
This is exactly as it came from apple, but I'm not sure if these empty 'g' elements are needed?